src/app/components/home/home-top-banner.component.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+
+.home-header-container {
+  display: flex;
+  background: linear-gradient($color-cerise-600, $color-primary-cerise);
+  color: $white;
+  flex-flow: row-reverse;
+  @media screen and (max-width: $md-device-width) {
+    flex-flow: column;
+  }
+
+  .home-header-map-image {
+    width: 426px;
+    padding-right: 0;
+    @media screen and (max-width: $md-device-width) {
+      margin: 0 auto;
+    }
+    @media screen and (max-width: 426px) {
+      width: 100%;
+    }
+  }
+
+  .home-header-info-container {
+    padding: 40px;
+    width: calc(100% - 426px);
+    text-align: left;
+    @media screen and (max-width: $md-device-width) {
+      padding-left: 10px;
+      padding-right: 10px;
+      width: auto;
+      max-width: 800px;
+      margin: 0 auto;
+      text-align: center;
+    }
+
+    .home-header-info-title {
+      color: $white;
+      text-shadow: 0 0 3px rgba(0, 0, 0, 0.75);
+      @include wrap-long-words;
```

Comment:
Expected item on line 41 to appear before line 39. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder)